packages/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -4436,14 +4436,17 @@ export default class Meeting extends StatelessWebexPlugin {
   };
 
   createMediaConnection(turnServerInfo) {
-    const mc = Media.createMediaConnection(this.mediaProperties, {
-      isMultistream: this.isMultistream,
-      meetingId: this.id,
-      remoteQualityLevel: this.mediaProperties.remoteQualityLevel,
-      enableRtx: this.config.enableRtx,
-      enableExtmap: this.config.enableExtmap,
-      turnServerInfo
-    });
+    const mc = Media.createMediaConnection(
+      this.isMultistream,
+      `MC-${this.id.substring(0, 4)}`,
```

Comment:
IMO the id should be done as a function, i.e.,

```
.../utils
const generateMeetingDebugId = (prefix = 'MC', id = 'DEFAULT') => return `${prefix}-id.substring(0, 4)`
...
and then call that function here
```

Reply:
done